source/_integrations/sleepiq.markdown

```diff
@@ -37,3 +37,52 @@ password:
   required: true
   type: string
 {% endconfiguration %}
+
+## Services
+
+SleepIQ exposes a few services to control your Sleep Number bed via automations:
+
+This changes a bed's sleep number to the specified value. This value should be a multiple of 5 between 5 and 100. Other values will work, but they will be rounded to the nearest multiple of 5 by the Sleep Number API.
+
+```yaml
+- service: sleepiq.set_sleep_number
+  data:
+    sleep_number: 45
+    bed: 'Master Bedroom'
+    side: 'right'
+```
+
+{% configuration %}
```

Comment:
Please do not use integration configuration block for service calls.

Check other parts of the documentation on how those are normally formatted.

Reply:
@frenck Addressed; I was looking at the Domino's component and accidentally thought their configuration was a service call.